src/main/java/org/apache/commons/text/translate/UnicodeUnpairedSurrogateRemover.java

```diff
@@ -31,12 +31,9 @@
      */
     @Override
     public boolean translate(final int codepoint, final Writer writer) throws IOException {
-        if (codepoint >= Character.MIN_SURROGATE && codepoint <= Character.MAX_SURROGATE) {
-            // It's a surrogate. Write nothing and say we've translated.
-            return true;
-        }
+        // It's a surrogate. Write nothing and say we've translated.
```

Comment:
@arturobernalg I think this comment here doesn't relate to the single return statement now (it is, and it is not, true and false now).

Maybe change it to `// If true, it is a surrogate. Write nothing and say we've translated. Otherwise return false, and don't translate it.`

Reply:
Agree. Changed.
Thank you